autopush-common/build.rs

```diff
@@ -1,5 +1,5 @@
 pub fn main() {
-    if !(cfg!(feature = "dynamodb") || cfg!(feature = "bigtable")) {
-        panic!("No database defined! Please compile with either `features=dynamodb` or `features=bigtable`");
+    if !cfg!(feature = "bigtable") {
```

Comment:
Note for Me:
I was debating that we should drop this check, but I can see us creating a dummy storage system in the future for testing purposes.
I still hold that `bigtable` should be included in the `default` feature set, since we have only one now.

Reply:
Ah good point, so update this to `default` eh?

Reply:
No, I'd leave it as `feature = "bigtable"` here. I'd just make  sure that both 
autoendpoint/Cargo.toml & 
autoconnect/Cargo.toml:

```toml
default  = ["bigtable"]
```

So that it gets picked up. (I'm guessing that `"emulator"` slipped into `autoconnect/Cargo.toml` due to a bad merge or something. It's semi-harmless, but should be removed for production code.